doc/api/child_process.markdown

```diff
@@ -432,10 +146,21 @@ the existing process and uses a shell to execute the command.*
   * `stderr` {Buffer}
 * Return: ChildProcess object
 
-This is similar to [`child_process.exec()`][] except it does not execute a
-subshell but rather the specified file directly. This makes it slightly
-leaner than [`child_process.exec()`][]. It has the same options.
+The `child_process.execFile()` method is similar to [`child_process.exec()`][]
+except that it does not first spawn a shell. Rather the specified `command` is
```

Comment:
and since it doesn't spawn a shell, cannot run node/npm installed .cmd wrappers for node scripts, such as npm!